content/tech-big-picture/security-best-practices-for-you/_index.md

```diff
@@ -7,10 +7,11 @@ prerequisites:
 ready: true
 title: Security best practices for you
 ---
+![best security practices](security.png)
```

Comment:
where is security.png? I'm not seeing it in the repo so this picture will be broken. Did you forget to add it to your PR?